genjava/client/okhttp_impl.go

```diff
@@ -49,8 +44,7 @@ func (g *Generator) client(api *spec.Api, apiPackage packages.Module, modelsVers
 	w.EmptyLine()
 	w.Line(`  public %s(String baseUrl) {`, className)
 	w.Line(`    this.baseUrl = baseUrl;`)
-	w.Line(`    this.objectMapper = new ObjectMapper();`)
-	w.Line(`    Json.setupObjectMapper(objectMapper);`)
+	g.Models.InitJsonMapper(w.IndentedWith(2))
```

Comment:
What about objectMapper field declared above in the client class? It also depends on JSON library....